runtime/permissions/mod.rs

```diff
@@ -39,15 +39,16 @@ pub use prompter::PromptCallback;
 static DEBUG_LOG_ENABLED: Lazy<bool> =
   Lazy::new(|| log::log_enabled!(log::Level::Debug));
 
-/// Tri-state value for storing permission state
+/// Quadri-state value for storing permission state
 #[derive(
   Eq, PartialEq, Default, Debug, Clone, Copy, Deserialize, PartialOrd,
 )]
 pub enum PermissionState {
   Granted = 0,
+  GrantedPartial = 1,
   #[default]
-  Prompt = 1,
-  Denied = 2,
+  Prompt = 2,
+  Denied = 3,
 }
```

Comment:
Is this enum's value somehow reachable to user code? Ie. will this be a breaking change from user code perspective that `GrantedPartial` has now value 1 instead of `Prompt`?

Reply:
I believe this is for serializing permissions between workers. It shouldn't leave process internals.